common/src/main/java/org/corfudb/common/metrics/micrometer/registries/dropwizard/DropwizardDistributionSummaryCustom.java

```diff
@@ -0,0 +1,58 @@
+package org.corfudb.common.metrics.micrometer.registries.dropwizard;
+
+import io.micrometer.core.instrument.AbstractDistributionSummary;
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.distribution.DistributionStatisticConfig;
+import io.micrometer.core.instrument.distribution.TimeWindowMax;
+import io.micrometer.core.instrument.util.MeterEquivalence;
+import io.micrometer.core.lang.Nullable;
+
+import java.util.concurrent.atomic.DoubleAdder;
+
+public class DropwizardDistributionSummaryCustom extends AbstractDistributionSummary {
+    private final com.codahale.metrics.Histogram impl;
```

Comment:
Do we want to import the drop wizard package? DropwizardTimerCustom does import but here it doesn't. Would be great to be consistent.

Reply:
fixed